x-pack/plugins/profiling/server/utils/create_profiling_es_client.ts

```diff
@@ -82,6 +83,27 @@ export function createProfilingEsClient({
         );
       });
 
+      return unwrapEsResponse(promise);
+    },
+    transport<TResponse = unknown>(
+      operationName: string,
+      transportRequest: any
+    ): Promise<TResponse> {
+      const controller = new AbortController();
+
+      const promise = withProfilingSpan(operationName, () => {
+        return cancelEsRequestOnAbort(
+          esClient.transport.request(transportRequest, {
+            signal: controller.signal,
+            meta: true,
+          }) as unknown as Promise<{
+            body: TResponse;
+          }>,
+          request,
+          controller
+        );
+      });
+
```

Comment:
@dgieselaar Does this look right? I tried to follow the types from `search` and `mget`. Specifically, I used `any` for `transportRequest` to satisfy the linter and compiler, but I wasn't sure if there was a more appropriate type.

Reply:
`transport` is very much a catch-all endpoint, so there's not much to type here. `unknown` would be slightly better than `any` because it requires the consumer of the endpoint to explicitly type it before it can be used. An alternative I'd suggest here is to not expose transport, but `proflingStacktraces` which _is_ typed. `transport` would just be an implementation detail. That allows us to use it in multiple places. I don't expect that we have to use `transport` for something else.